core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -530,6 +531,17 @@ public void invoke() {
         }
     }
 
+    // #460 : docker-compose are not resolvable on Win10 machines because there is docker-compose.exe binary,
+    // but not docker-compose
+    private boolean isDockerComposeExists() {
```

Comment:
I would prefer renaming the method to something like `dockerComposeExecutableExists`.